### PR TITLE
Add DataImportCron snapshot sources docs

### PR DESCRIPTION
**Pull request by akalenyu**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: doc/os-image-poll-and-update.md [hunk 1]
```diff
@@ -84,4 +84,17 @@ Or on CRC:
...
-More information on image streams is available [here](https://docs.openshift.com/container-platform/4.8/openshift_images/image-streams-manage.html) and [here](https://www.tutorialworks.com/openshift-imagestreams).
\ No newline at end of file
+More information on image streams is available [here](https://docs.openshift.com/container-platform/4.8/openshift_images/image-streams-manage.html) and [here](https://www.tutorialworks.com/openshift-imagestreams).
+
+## DataImportCron source formats
+
+* PersistentVolumeClaim
+* VolumeSnapshot
+
+DataImportCron has only maintained PVC sources for a long period of time,
```
Comment (arnongilboa) [inline doc/os-image-poll-and-update.md:94]: maybe: DataImportCron was originally designed to work with PVC sources,

File: doc/os-image-poll-and-update.md [hunk 2]
```diff
@@ -84,4 +84,17 @@ Or on CRC:
...
\ No newline at end of file
+More information on image streams is available [here](https://docs.openshift.com/container-platform/4.8/openshift_images/image-streams-manage.html) and [here](https://www.tutorialworks.com/openshift-imagestreams).
+
+## DataImportCron source formats
+
+* PersistentVolumeClaim
+* VolumeSnapshot
+
+DataImportCron has only maintained PVC sources for a long period of time,  
+However, for certain storage types, we know that snapshots provide a better fit.
```
Comment (arnongilboa) [inline doc/os-image-poll-and-update.md:95]: fit -> scaling?

File: doc/os-image-poll-and-update.md [hunk 3]
```diff
@@ -84,4 +84,17 @@ Or on CRC:
...
+More information on image streams is available [here](https://docs.openshift.com/container-platform/4.8/openshift_images/image-streams-manage.html) and [here](https://www.tutorialworks.com/openshift-imagestreams).
+
+## DataImportCron source formats
+
+* PersistentVolumeClaim
+* VolumeSnapshot
+
+DataImportCron has only maintained PVC sources for a long period of time,  
+However, for certain storage types, we know that snapshots provide a better fit.  
+Some examples and more info can be found in [clone-from-volumesnapshot-source](./clone-from-volumesnapshot-source.md).
```
Comment (arnongilboa) [inline doc/os-image-poll-and-update.md:96]: See details and examples here

File: doc/os-image-poll-and-update.md [hunk 4]
```diff
@@ -84,4 +84,17 @@ Or on CRC:
...
+
+We keep this provisioner specific knowledge on the [StorageProfile](./storageprofile.md) object for each provisioner under `dataImportCronSourceFormat` (possible values are `snapshot`/`pvc`), which in turns signals the data import cron about which type of source is preferred for a specific provisioner.
```
Comment (arnongilboa) [inline doc/os-image-poll-and-update.md:98]: We keep this provisioner-specific information in the [StorageProfile](./storageprofile.md) of the provisioner in `dataImportCronSourceFormat` (possible values are `snapshot`/`pvc`), which tells the DataImportCron which type of source is preferred for the provisioner.

File: doc/os-image-poll-and-update.md [hunk 5]
```diff
@@ -84,4 +84,17 @@ Or on CRC:
 * oc import-image cirros-is -n openshift-virtualization-os-images --from=kubevirt/cirros-container-disk-demo --scheduled --confirm
 * oc set image-lookup cirros-is -n openshift-virtualization-os-images
 
-More information on image streams is available [here](https://docs.openshift.com/container-platform/4.8/openshift_images/image-streams-manage.html) and [here](https://www.tutorialworks.com/openshift-imagestreams).
\ No newline at end of file
+More information on image streams is available [here](https://docs.openshift.com/container-platform/4.8/openshift_images/image-streams-manage.html) and [here](https://www.tutorialworks.com/openshift-imagestreams).
```
Comment (apinnick) [inline doc/os-image-poll-and-update.md:87]: Suggest updating this link: https://docs.openshift.com/container-platform/4.13/openshift_images/image-streams-manage.html

File: doc/os-image-poll-and-update.md (hunk 1 above)
Comment (akalenyu) [reply doc/os-image-poll-and-update.md]: done

File: doc/os-image-poll-and-update.md (hunk 2 above)
Comment (akalenyu) [reply doc/os-image-poll-and-update.md]: done

File: doc/os-image-poll-and-update.md (hunk 3 above)
Comment (akalenyu) [reply doc/os-image-poll-and-update.md]: rewrote

File: doc/os-image-poll-and-update.md (hunk 4 above)
Comment (akalenyu) [reply doc/os-image-poll-and-update.md]: :+1:

File: doc/os-image-poll-and-update.md (hunk 5 above)
Comment (akalenyu) [reply doc/os-image-poll-and-update.md]: done, thx

**Comment by akalenyu** [comment]
/retest
@arnongilboa ping

**Pull request merged**